src/GeneralStateTestsFiller/VMTests/vmTests/push0Filler.yml

```diff
@@ -0,0 +1,185 @@
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        17171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717171717\
+        60019055"
+      nonce: '0'
+      storage: {}
+
+    0000000000000000000000000000000000000300:
+      balance: 0
+      # 1025 push0's
+      code: ":raw 0x\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
```

Comment:
what are you trying to break by calling it 1025 times ?

Reply:
> what are you trying to break by calling it 1025 times ?

This case comes from https://eips.ethereum.org/EIPS/eip-3855#test-cases
I think it checks that exactly 1 item is pushed on stack, so 1025 of them overflow stack.

Reply:
> > also make sure it does not mess up stack if there is some stack pushed already and then push0 happens.
> 
> how would you check this?

init:
store: { 01 :  10,  00 : 10 }
code:  60015f55

after the exuction the corresponding cell must change to 0